### PR TITLE
Add documentation on PR agreements

### PR DESCRIPTION
**Pull request by simonrondelez**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by iuriaranda — COMMENTED** [review]
I think the content itself is good, but I'm not sure about the location.

What about a CONTRIBUTING.md file in the root of the repo, this also matches the open source style.

File: coding_guidelines/git.md [hunk 1]
```diff
@@ -14,12 +14,22 @@
      * If applied, this commit will _Update survey-csv worker to queue:work (#396)_
      * If applied, this commit will _Remove deprecated methods_
      * If applied, this commit will _Deploy dashboard and OIDC proxies skyscrapers/runtime#3_
-* Optional: Add a reference (GH issue) to your message. This automatically links your commit back to an issue. This is preferred when commiting something to `master`
+* Optional: Add a reference (GH issue) to your message. This automatically links your commit back to an issue. This is preferred when commiting something to the `main` branch
 * Finally, make sure to commit correct file permissions (especially for Windows users 😉): `644` for all files unless it's meant to be an executable (eg. `skysginin`, `755`)
 
+## Pull requests
+
+We highly endorse you to make contributions that you are comfortable with. In order to work together on the same codebase we have some ground rules:
```
Comment (fdenkens) [inline coding_guidelines/git.md:22]: `endorse` is misused here. I suggest using the word `encourage`

**Comment by simonrondelez** [comment]
i wasn't sure myself about the location. I feel a `CONTRIBUTING.md` might be too confusing as that usually only affects that repo. Maybe we should add it to the template? 🤔

File: coding_guidelines/git.md [hunk 2]
```diff
@@ -14,12 +14,22 @@
...
+
+* All code changes should be tested, documented and follow the style of that project.
+* Code changes should be pushed to a branch and created into a pull request that is reviewed by someone at Skyscrapers.
+* A pull request can only be rolled out when it is approved.
+* When a pull request gets approved it should be applied before merging to the main branch (*Note:* don't forget to rebase the main branch before applying the changes to avoid rolling back any changes that were done before the creation of the branch).
```
Comment (fdenkens) [inline coding_guidelines/git.md:27]: Line 26 and 27 both talk about it being approved = duplication.

**Review by fdenkens — APPROVED** [review]
General remark: I just focused on wording and sentences. Not functional flow. 
Also: seems more like it is written for our own team's pull requests rather than for customers. But I could be wrong and missing context.

**Review by duboisph — APPROVED** [review]
Lgtm.

I think it's fine to put here, but also like @simonrondelez 's remark of adding a `CONTRIBUTING` file to our repo template.

**Comment by iuriaranda** [comment]
The way I see it, all documentation in the customer repo should be specific to the customer, as generic documentation is hard to manage and maintain when it's replicated, and that's why we put it in this repo.

True that a `CONTRIBUTING.md` file in this repo could be confusing, but imo we could clear out that confusion within the file, and on the plus side it would give it more visibility than putting it here in the git guidelines.

That being said, I don't object to the PR as it is now, but my concern is that these guidelines here won't be visible enough and will most probably go unnoticed by most customers.

**Comment by fdenkens** [comment]
I may chime in: 

1) Company guideline: deduplicate as much as possible. Hence generic, centralised documentation is preferred.
2) A reference in each repo to the central documentation certainly won't hurt.
3) Although this process is indeed maybe a but 'hidden', that is because it is more documented as 'reference' documentation. I think that is fixable by other forms of communication that over time can complement this: leads making their customers aware, new more practical documentation on how to work together, etc.

2 and 3 are just ideas that may help. Do with them what you want.

**Comment by simonrondelez** [comment]
i feel that both a CONTRIBUTING file as here in the repo can be missed / ignored equally 🤔 . Its just we that need to inform our customers about it. Hence i added it here to the other things our customers should be aware of.

**Comment by simonrondelez** [comment]
> 2. A reference in each repo to the central documentation certainly won't hurt.

This is already the case

**Comment by simonrondelez** [comment]
For now lets keep it like this. I'll document this process in the .github repo so it is in the template of the PR when creating.

**Pull request merged**